pkg/server/commands/list_objects_test.go

```diff
@@ -22,3 +31,141 @@ func TestNewListObjectsQuery(t *testing.T) {
 		require.Error(t, err)
 	})
 }
+
+func TestListObjectsDispatchCount(t *testing.T) {
+	ds := memory.New()
+	ctx := storage.ContextWithRelationshipTupleReader(context.Background(), ds)
+
+	tests := []struct {
+		name                  string
+		model                 string
+		tuples                []*openfgav1.TupleKey
+		objectType            string
+		relation              string
+		user                  string
+		expectedDispatchCount uint32
+	}{
+		{
+			name: "test_direct_relation",
+			model: `model
+schema 1.1
```

Comment:
Here are below, can we please format these models so they are indented inline with the field that defines them?

Reply:
I tried to, was getting some model parsing errors. Saw check had same way of implementing these so just kept it. Let me check for someother example

Reply:
Historically the model parser was finicky about whitespace. So that's why you see old references to models with poor indentation. I thought we updated our model parser to the latest one, which isn't so picky about whitespace.. Maybe not though? Please confirm.

Reply:
Well looks like we need schema to be the first line with no spaces or empty lines above. I don't know whether that is necessary